lib/oaken/seeds.rb

```diff
@@ -20,22 +20,39 @@ def self.register(type, key = nil)
   end
   def self.provider = Oaken::Stored::ActiveRecord
 
-  singleton_class.attr_reader :loader
-  delegate :entry, to: :loader
+  class << self
+    # Expose a class `seed` method for individual test classes to use.
+    # TODO: support parallelization somehow.
```

Comment:
So I think this approach actually is the best way forward for supporting parallelization. My reason being that it's called on a per-test basis. Tests are already broken up into individual units of work from a parallelization perspective. Having the `seed` method be called on a per-test basis (or in a `setup`/`before` block, which is also called on an individual unit of work) helps reinforce test isolation. That:

- Reduces the amount of junk data that's generated for unrelated tests
- Makes it easier to debug a particular test
- Reduces test flakiness
- Encourages writing seed files for specific edge-case scenarios

Reply:
@tcannonfodder I added this as a recommendation and explanation in https://github.com/kaspth/oaken/commit/6ed2503b2a4453ccb861a21c725615cb28b7d0c1